src/main/java/org/jenkinsci/plugins/github_branch_source/GitHubSCMSource.java

```diff
@@ -1005,7 +1005,12 @@ public GHPermissionType fetch(String username) throws IOException, InterruptedEx
                                         public SCMSourceCriteria.Probe create(@NonNull BranchSCMHead head,
                                                                               @Nullable SCMRevisionImpl revisionInfo)
                                                 throws IOException, InterruptedException {
-                                            return new GitHubSCMProbe(github, ghRepository, head, revisionInfo);
+                                            return new GitHubSCMProbe(
```

Comment:
GitHubSCMProbe will close the `GitHub` instance passed to it when the probe is closed.  Passing in an existing instance could result in an extra close call to that instance.